test/e2e/init_scale_zero_test.go

```diff
@@ -128,20 +173,10 @@ func TestInitScaleZeroMinScaleClusterLevel(t *testing.T) {
 	}
 	defer test.TearDown(clients, names)
 
-	t.Log("Fetch config-defaults ConfigMap")
-	original, err := rawCM(clients, "config-defaults")
-	if err != nil || original == nil {
-		t.Fatalf("Error fetching config-defaults: %v", err)
-	}
-	var newCM *v1.ConfigMap
-	original.Data["check-validity-on-deploy"] = "false"
-	if newCM, err = patchCM(clients, original); err != nil {
-		t.Fatalf("Failed to update config defaults: %v", err)
-	}
-	t.Logf("Successfully updated configMap.")
-	newCM.Data["check-validity-on-deploy"] = "true"
-	test.CleanupOnInterrupt(func() { restoreClusterDefaults(t, clients, newCM) })
-	defer restoreClusterDefaults(t, clients, newCM)
+	updatedConfigDefaultsCM := setCheckValidityOnDeployOnCluster(t, clients, "config-defaults", false)
+	updatedConfigAutoscalerCM := setCheckValidityOnDeployOnCluster(t, clients, "config-autoscaler", false)
```

Comment:
we should probably have one function to deal with these and only pass the name. Also we can get the config name from the constant that is associated.

Reply:
yeah we actually don't have to pass in the true or false, since we only call this when we want to set the flag to false